app/src/main/res/layout/fragment_chat.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/cl_chat_list_parent_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center_horizontal"
+    android:paddingEnd="@dimen/activity_horizontal_padding"
+    android:paddingStart="@dimen/activity_horizontal_padding">
+
+
+    <android.support.v7.widget.SearchView
+        android:id="@+id/sv_fragment_chat_search_people"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="@dimen/input_margin_start_to_end"
+        android:layout_marginStart="@dimen/input_margin_start_to_end"
+        android:layout_marginTop="@dimen/margin_16_dp"
```

Comment:
having the dimensions in the name, I don't think is a good naming convention... what will be the point of separating the measure and the name of the "constant" if the name of the constant will have the value...